subcommittee/tooling/compiler-features/desired-compiler-features.yaml

```
@@ -0,0 +1,84 @@
+# This file lists desired Rust compiler features for security and safety-critical applications,
+# which are currently missing or incomplete in the standard Rust toolchain.
+
+metadata:
+  title: "Desired Rust Compiler Features"
+  version: "1.0"
+  date: "2025-10-01"
+  purpose: "Track missing compiler features needed for safety-critical Rust applications"
+  considered-standards:
+    - name: "ISO-26262"
```

The standard has a space here
```suggestion
    - name: "ISO 26262"
```